doc/rst/sw_pdm.rst

```diff
@@ -11,16 +11,17 @@ The PDM stream from the microphones is converted to PCM and output to the host v
 Interfacing to the PDM microphones is done using the `XMOS` microphone array library
 (`lib_mic_array <https://www.xmos.com/file/lib_mic_array>`_).
 ``lib_mic_array`` is designed to allow interfacing to PDM microphones coupled to efficient decimation filters
-at a user configurable output sample rate. Currently dynamic sample rate changing is not supported.
+at a user configurable output sample rate.
 
 .. note::
     The ``lib_mic_array`` library is only available for `xcore.ai` series devices since it uses the
     Vector Processing Unit of the XS3 architecture.
 
-Up to eight PDM microphones can be attached the PDM interface (``mic_array_task()``) but it is
+Up to eight PDM microphones can be attached to the PDM interface (``mic_array_task()``) but it is
 possible to extend this.
 
-After PDM capture and decimation to the output sample-rate various other steps take place e.g. DC offset elimination etc. Please refer to the documentation provided with  ``lib_mic_array`` for further implementation detail and a complete feature set.
+After PDM capture and decimation to the output sample-rate various other steps take place e.g. DC offset elimination etc.
+Please refer to the documentation provided with  ``lib_mic_array`` for further implementation detail and a complete feature set.
```

Comment:
link to https://www.xmos.com/libraries/lib_mic_array would be nice here.

Reply:
done